apps/crn-frontend/src/events/state.ts

```diff
@@ -117,6 +117,10 @@ export const useEvents = (options: GetEventListOptions, user?: User | null) => {
   const [events, setEvents] = useRecoilState(eventsState(options));
   const { client } = useAlgolia();
 
+  if (user && !user.onboarded) {
```

Comment:
If it's _possible_ that `user` is not defined then surely we'd want to treat that case as the same as a non-onboarded user?
```suggestion
  if (!user || !user.onboarded) {
```

Reply:
We use this hook in different components where we don't send the user as the 2nd param